integrationTests/bootstrap.php

```diff
@@ -36,6 +38,8 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
     }
 });
 
+$container->set(TransportInterface::class, new InmemoryTransport());
```

Comment:
```suggestion
$container->set(TransportInterface::class, new InMemoryTransport());
```
Presumably harmless as tests pass, but suggesting just in case given the weird Doctrine thing around case recently

Reply:
Thanks, not sure how the autoloading on Linux was able to work with the wrong case, I thought it was a case-sensitive filesystem.